idaes/core/util/tests/test_model_diagnostics.py

```diff
@@ -975,7 +975,9 @@ def test_display_near_parallel_variables(self):
         expected = """====================================================================================
 The following pairs of variables are nearly parallel:
 
+    v1, v2
```

Comment:
Is this correct? I would not have thought v1 and v2 were (nearly) parallel for this example.

Reply:
In a relative sense, they are. By my math, they have Jacobian columns:
```
v1: (1, 1, 1e8, -1)
v2: (-1, 0, 1e10, 1-1e5)
```
Within our tolerance of 1e-4, all entries except 1e8 and 1e10 are very small, so the dot product of these two vectors is very close to the product of the norms, relative to the size of the two vectors.